pandas/tests/tseries/test_ticks.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for offsets.Tick and subclasses
+"""
+from datetime import datetime, timedelta
+
+import pytest
+import numpy as np
+
+from pandas import Timedelta, Timestamp
+from pandas.tseries import offsets
+from pandas.tseries.offsets import Hour, Minute, Second, Milli, Micro, Nano
+
+
+# ---------------------------------------------------------------------
+# Test Helpers
+
+tick_classes = [Hour, Minute, Second, Milli, Micro, Nano]
+
+
+def assertEq(offset, base, expected):
+    actual = offset + base
+    actual_swapped = base + offset
+    actual_apply = offset.apply(base)
+    try:
+        assert actual == expected
+        assert actual_swapped == expected
+        assert actual_apply == expected
+    except AssertionError:
+        raise AssertionError("\nExpected: %s\nActual: %s\nFor Offset: %s)"
+                             "\nAt Date: %s" %
+                             (expected, actual, offset, base))
+
+# ---------------------------------------------------------------------
+
+
+def test_apply_ticks():
+    result = offsets.Hour(3).apply(offsets.Hour(4))
+    exp = offsets.Hour(7)
+    assert (result == exp)
+
+
+def test_delta_to_tick():
+    delta = timedelta(3)
+
+    tick = offsets._delta_to_tick(delta)
+    assert (tick == offsets.Day(3))
+
+
+# ---------------------------------------------------------------------
+
+
+class TestTicks(object):
```

Comment:
I don't really like classes to be honest, rather than them at the top-level. I would only do this if you have a small number of different things you are testing that don't warrant a module each

Reply:
I tend to agree.  For now I'm just separating these out because Ticks are going to require a bunch of separate treatment (and the test_offsets file is pretty unwieldy at ~4500 lines)

Reply:
again, instead of doing this, just make a another level and put things in modules, then all happy, e.g.

``pandas/tests/tseries/offsets/test_*`` is simplea and clear (and you can even push the conftest to this directly to make it even more obvious).

Reply:
OK.  Note that the TestTick class is not my doing, is just moved.